index.html

```diff
@@ -35,6 +35,125 @@ <h2 class="greeting-intro">
             <li><a href=""><img class="social" src="assets/img/github.png" alt="github"/></a></li>
             <li><a href=""><img class="social" src="assets/img/angellist.png" alt="angellist"/></a></li>
         </ul>
+        <button class="scroll-down">
+            <img class="disabled-button" src="assets/img/Disabled.png" alt="disabled-button">
+        </button>
+    </section>
+    <section class="recent-works-title">
+        <h2 class="home-headings works-heading">My Recent Works</h2>
+    </section>
+    <section class="works-wrapper">
+        <section class="works-card">
+            <section class="works-card-inner">
```

Comment:
Kindly. the semantic SECTION tags are used to identify each theme of your page, I recommend you use them like this, inside them you can put the necessary content, but putting a SECTION inside another SECTION is not a good practice, I recommend changing it to a DIV.

Reply:
This is noted. Thanks for correction 👍